index.js

```diff
@@ -1,102 +1,106 @@
 /**
- * Винести змінні які ти мутуєш в окремий об'єкт як ключі
- * Пройтись по змінних і замінити let на const у місцях де це можливо
- * Змінні в camelCase
- * DRY principle: накнайменше дублікацій коду
- * Переписати проміси на async/await
+ * Винести змінні які ти мутуєш в окремий об'єкт як ключі +
+ * Пройтись по змінних і замінити let на const у місцях де це можливо +
+ * Змінні в camelCase +
+ * DRY principle: накнайменше дублікацій коду +
+ * Переписати проміси на async/await +
  * */
 
 
 (function (){
   const pagination = {
-
+    page: 1,
+    perPage: 2,
+    total: document.getElementById('total'),
+    totalCount: 0,
+    pageNumberSpan: document.getElementById('page-number'),
+    pageNumber: 1,
+    btnNext: document.getElementById('btn_next'),
+    btnPrev: document.getElementById('btn_prev'),
+    url: 'https://reqres.in/api/users?'
   }
-  let page = 1;
-  let per_page = 2;
-  let total = document.getElementById('total');
-  let total_count = 0;
-  let page_number_span = document.getElementById('page-number');
-  let page_number = 1
-  let btn_next = document.getElementById('btn_next');
-  let btn_prev = document.getElementById('btn_prev');
-  const url = 'https://reqres.in/api/users?';
 
 // Send request and work with data.
-  function load() {
-    fetch(url + new URLSearchParams({
-      page: page,
-      per_page: per_page,
-    }))
-        .then(function (respose) {
-          return respose.json()
-        })
-        .then(function (responseData) {
-          total_count = responseData.total;
-          // Render total results.
-          total.innerHTML = `${per_page} of ${responseData.total} Records`;
-
-          let result = document.querySelector('#result');
-          // Clear result.
-          result.innerHTML = ""
-
-          // load responce data.
-          let data = responseData.data
-
-          let html = data.map(function(item) {
-            // here we need to generate html for object.
-            return `<div class="user-info"><div class="user-first-name">${item.first_name}</div><div class="user-last-name">${item.last_name}</div><div class="user-email">${item.email}</div><div class="user-avatar"><img src="${item.avatar}"></div></div>`
-          })
-
-          result.insertAdjacentHTML('afterbegin', html.join(''))
-        })
-  }
+async function load() {
+  const response = await fetch(pagination.url + new URLSearchParams({
```

Comment:
We can create a variable for the URL: `const fetchURL = pagination.url + new URLSearchParams({page: pagination.page,`
    `per_page: pagination.perPage,`
`  })`